editoast/src/generated_data/sql/generate_psl_sign_layer.sql

```diff
@@ -30,36 +33,79 @@ WITH signs AS (
 collect AS (
     SELECT signs.sc_id,
         signs.data,
+        tracks_layer.geographic AS track_geo,
+        tracks_layer.schematic AS track_sch,
+        LEAST(
+            GREATEST(position / (tracks.data->>'length')::float, 0.),
+            1.
+        ) AS norm_pos,
         ST_LineInterpolatePoint(
             tracks_layer.geographic,
             LEAST(
-                GREATEST(
-                    signs.position / (tracks.data->'length')::float,
-                    0.
-                ),
+                GREATEST(position / (tracks.data->>'length')::float, 0.),
                 1.
             )
```

Comment:
Can't you use `norm_pos` instead of these 4 lines? It looks like the code from https://github.com/osrd-project/osrd/pull/6481/files#diff-d93d07d2bf101b6630091e9b4132bcc89aaac24d3ee1fae6c67a609d097efc24R38-R41 is exactly the same 🤔

Reply:
yes indeed ^^